msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
+
+#### Configure Adaptive Card responsive layout
+
+The following are the key elements to create a responsive Adaptive Card:
+
+* `CardElement.targetWidth`: Allows you to control the visibility of any element by linking it to `hostWidth`. Add the `targetWidth` property for all elements in an Adaptive Card to define different layouts for different card widths. The following are the predefined values for the `targetWidth` property:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `very narrow` | The `targetWidth` for right narrow pane in the meeting chat. |
+  | `narrow` | The `targetWidth` for mobile screen. |
+  | `standard` | The `targetWidth` for chats and channels in desktop. |
+  | `wide` | The `targetWidth` for any screen size beyond `standard`. |
+
+  The values for `targetWidth` refers to the relative width of the card and allows more flexibility in adapting to different screen sizes.
+
+  > [!NOTE]
+  > We recommend to use the predefined values for the `targetWidth` property for your Adaptive Card and not the pixel sizes of screen.
+
+* `CardElement.hostWidth`: Allows the host application to define the width it provides to the card through `targetWidth` property. The host application is responsible for mapping the actual widths to these named values. The following are the predefined values for the `hostWidth` property:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `very narrow` | The `hostWidth` for right narrow pane in the meeting chat. |
+  | `narrow` | The `hostWidth` for mobile screen. |
+  | `standard` | The `hostWidth` for chats and channels in desktop. |
+  | `wide` | The default value for `hostWidth`. |
+
+  You can add the `targetWidth` property to any element with a prefix to the predefined value that is `atLeast` or `atMost`. When you specify the targetWidth value to a predefined value with a prefix, the elements are displayed or hidden based on the `hostWidth`, providing you with the control over layout responsiveness.
+
+  The following examples explain how the elements can be made visible or hidden based on the condition with `targetWidth` and `hostWidth`: 
+
+  * If you set `targetWidth` to `standard`, the element is only visible when the `hostWidth` property is set to `wide`.
+
+  * If you set `targetWidth` to `atMost:narrow`, the element is only visible if the `hostWidth` property is set to `veryNarrow` or `narrow`.
+
+  * If you set `targetWidth` to `atLeast:standard`, the element is only visible if the `hostWidth` property is set to `standard` or `wide`.
+
+#### Sample Adaptive Card responsive layout
+
+To make a responsive Adaptive Card, your app must include the elements from the following code sample:
```

Comment:
Coul you please add screenshots of that example card at different widths so as to show the reader the actual effect of using the `targetWidth` property?